.github/workflows/issue-auto-pr.yml

```
@@ -0,0 +1,54 @@
+        uses: actions/setup-node@v4
+        with:
+          node-version: 'node'
+
+      - name: Install Codex
+        run: npm install -g @openai/codex
+
+      - name: Set environment variable for OpenAI key
+        run: echo "OPENAI_API_KEY=${{ secrets.OPENAI_KEY }}" >> $GITHUB_ENV
+
+      - name: Run Codex with issue text
+        run: |
```

actionlint (1.7.7)

"github.event.issue.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)